tests/foreman/cli/test_container_management.py

```diff
@@ -322,3 +322,51 @@ def test_positive_container_admin_end_to_end_pull(
         # 9. Pull in docker image
         result = container_contenthost.execute(docker_pull_command)
         assert result.status == 0
+
+    def test_negative_pull_content_with_longer_name(target_sat, default_org):
+        """Verify that long name CV publish when CV & docker repo both have larger name.
+
+        :id: e0ac0be4-f5ff-4a88-bb29-33aa2d874f46
+
+        :steps:
+
+            1. Create Product,docker repo,CV and LCE with long name product / repo
+            2. Sync the repos
+            3. Add repository to CV, Publish, Promote CV to LCE
+            4. Pull in docker image
+
+        :expectedresults:
+
+            1. Long Product, repository, CV and LCE should create successfully
+            2. Sync repository successfully
+            3. Publish & Promote should success
+            4. Can pull in docker images
+
+        :BZ: 2127470
+
+        :customerscenario: true
+        """
+        pattern_postfix = gen_string('alpha', 50)
+
+        product_name = f'Containers-{pattern_postfix}'
+        repo_name = f'Repo-{pattern_postfix}'
+        lce_name = f'LCE-{pattern_postfix}'
+        cv_name = f'LCE-{pattern_postfix}'
+
+        logger.info("Creating product with longer name")
+        product = make_product_wait({'name': product_name, 'organization-id': default_org.id})
+        logger.info("Creating docker repo with longer name")
+        repo = _repo(product['id'], name=repo_name, upstream_name=CONTAINER_UPSTREAM_NAME)
+        Repository.synchronize({'id': repo['id']})
```

Comment:
```suggestion
        target_sat.cli.Repository.synchronize({'id': repo['id']})
```
...and lower for the entities make and CV calls too.